src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -295,24 +300,20 @@ public IList<string> Merge (TaskLoggingHelper log, TypeDefinitionCache cache, Li
 					app.Add (new XElement ("meta-data", new XAttribute (androidNs + "name", "com.google.android.wearable.beta.app"), new XAttribute (androidNs + "resource", "@xml/wearable_app_desc")));
 			}
 
-			// If no <uses-sdk> is specified, add it with both minSdkVersion and
-			// targetSdkVersion set to TargetFrameworkVersion
+			string minSdkVersionString = TryParseMinSdkVersionName ();
```

Comment:
This feels unnecessary, in that the only way `TryParseMinSdkVersionName()` will fail is if `MinSdkVersionName` contains an invalid value, and if that's the case surely we would have errored out elsewhere already?